zebROS_ws/src/ros_control_boilerplate/config/robot_in_a_box_jetson.yaml

```diff
@@ -30,6 +30,7 @@ hardware_interface:
        - {name: robot_code_ready_rio, local: false, type: ready} # Probably not really needed?
        - {name: do1, local: false, type: digital_output}
        - {name: do2, local_update: true, local_hardware: false, type: digital_output}
+       - {name: four_bar_leader, type: can_talon_fx, can_bus: CANivore, can_id: 22, local: true}
```

Comment:
This should be added to the 2023_combot_base_jetson.yaml file as well.  
Pick an appropriate CAN id, when we set up the hardware we can make it match up later.